package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
+    "semantic-release": "24.2.3",
+    "@semantic-release/git": "10.0.1",
+    "semantic-release-replace-plugin": "1.2.7",
+    "@hexonet/semantic-release-whmcs": "5.0.58"
```

Comment:
nit: 5.0.59 was released a few hours ago. Nothing major though. https://github.com/centralnicgroup-opensource/rtldev-middleware-semantic-release-whmcs/releases/tag/v5.0.59

Reply:
Yeah, those were actually my changes :) https://github.com/centralnicgroup-opensource/rtldev-middleware-semantic-release-whmcs/pull/401

Reply:
We could leave this up for dependabot to take care of :)

Reply:
Lets do it!

Reply:
https://github.com/UpCloudLtd/upcloud-whmcs-module/pull/74